test/functionalTest/cases/2995_q_in_GET/mq_compound_does-not-exist_GET_GTE.test

```diff
@@ -0,0 +1,144 @@
+# Copyright 2017 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
Extensive set of .test. I'm look then in detail, but they look great!

NTC

Reply:
Yeah, it was  quicker than I thought initially so I decided to add the entire 'matrix' of combinations